### PR TITLE
INTG-28: Document new features

### PR DESCRIPTION
**Pull request by JuhoHiltunen**

### CODE REVIEW

File: mule-bpm-module/README.md [hunk 1]
```diff
@@ -74,6 +74,14 @@ The core BPM engine implementation used by this Mule BPM Module. It is not curre
 
 ## Features
 
+### UniqueDeploymentName
+
+When 'duplicateDeploymentFiltering' is set to true and 'uniqueDeploymentName' is provided,
```
Comment (janihaglund) [inline mule-bpm-module/README.md:79]: Add description of result of setting this true: prevents deploying same process and form definitions, if there are no changes

File: mule-bpm-module/README.md [hunk 2]
```diff
@@ -104,6 +112,40 @@ Process factory allows adding variables to process, by using nested `<bpm:proces
 
 By setting `uniqueBusinessKey` attribute, the instance is guaranteed to be the only instance carrying the key.
 
+
+### Process querying & deletion
+
+Processes can be queried and deleted from Java.
```
Comment (janihaglund) [inline mule-bpm-module/README.md:118]: Add example of XML API usage

**Review by janihaglund — CHANGES_REQUESTED** [review]
Added couple missing documentations and Mermaid sequence diagrams

File: mule-bpm-module/README.md (hunk 1 above)
Comment (JuhoHiltunen) [reply mule-bpm-module/README.md]: [888b027](https://github.com/alfameCom/mule-bpm-module/commit/888b027bdaec556141d1e0863c9d29ee758a8b19)

File: mule-bpm-module/README.md (hunk 2 above)
Comment (JuhoHiltunen) [reply mule-bpm-module/README.md]: https://github.com/alfameCom/mule-bpm-module/commit/c9bacbfba9158c75e6e5a9e02b99ea043671a895

**Review by tuomasmakinen — APPROVED** [review]

**Comment by JuhoHiltunen** [comment]
New PR: https://github.com/alfameCom/mule-bpm-module/pull/66

This will be closed because of broken commit history

**Pull request closed without merging**